src/rt/CMakeLists.txt

```diff
@@ -5,7 +5,7 @@ set(SNMALLOC_ONLY_HEADER_LIBRARY ON)
 add_subdirectory(external/snmalloc EXCLUDE_FROM_ALL)
 
 macro(subdirlist result curdir)
-  file(GLOB children LIST_DIRECTORIES true RELATIVE ${curdir} ${curdir}/*)
+  file(GLOB children LIST_DIRECTORIES true CONFIGURE_DEPENDS RELATIVE ${curdir} ${curdir}/*)
```

Comment:
IIUC, this will update on `ninja` time if the files / dirs change, which it doesn't do now, right?

Though, according to the docs, it's not supported on all generators, so we probably need to be careful with that. However, CI will always catch it.

Reply:
Yep,  It seems a good thing to put in.  If it is not supported, then you will need to run cmake more.  Mostly, while reading docs I found it, and decided we should use it.  As several common issues will go away.